test.sh

```diff
@@ -7,4 +7,6 @@ IMG="$REGISTRY/$REPOSITORY:$TAG"
 echo "Running bats tests"
 docker run -i --entrypoint "bats" "$IMG" "/tmp/test"
 
+./test-auth.sh ${IMG} ${TAG}
```

Comment:
Can this gets quotes? This is a trivial case that gets called out by Shellcheck.

Can you please make sure it is installed and integrated with whatever editor you are using here?